### PR TITLE
feat: observer merge plugins

### PR DESCRIPTION
**Pull request by 0xff-dev**
#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
observer merge plugins.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: observer-plugins/default-plugins/pkg/plugins/prometheus/server.go [hunk 1]
```diff
@@ -101,14 +106,21 @@ func (p *prometheusServer) GetMetrics(ctx context.Context, req *obi.GetMetricsRe
...
 
 	klog.Infof("query by metric '%s', query '%s' successfully", req.MetricName, req.Query)
 	klog.V(5).Infof("%s query by %s, %s result: %v\n", method, req.MetricName, req.Query, metricData)
 
 	return result, nil
 }
+
+func init() {
+	cfg, err := clientcmd.BuildConfigFromFlags("", *flags.Kubeconfig)
+	klog.Infoln("----> ", *flags.Address)
```
Comment (nkwangleiGIT) [inline observer-plugins/default-plugins/pkg/plugins/prometheus/server.go:118]: Remove or refine log message?

File: observer-plugins/default-plugins/main.go [hunk 2]
```diff
@@ -17,70 +17,43 @@ limitations under the License.
...
-	)
-	if *kubeConfig != "" {
-		conf, err = clientcmd.BuildConfigFromFlags("", *kubeConfig)
-	} else {
-		conf, err = rest.InClusterConfig()
+	_, err := os.Stat(*flags.Endpoint)
+	if err != nil && !os.IsNotExist(err) {
+		klog.Fatalln(err)
 	}
+	klog.Infof("remove endpoint path: %s\n", *flags.Endpoint)
```
Comment (nkwangleiGIT) [inline observer-plugins/default-plugins/main.go:44]: No need to print this log or use low level log?

File: observer-plugins/default-plugins/staging/github.com/kube-arbiter/arbiter/pkg/proto/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+### protoc installation
+
+Enter the protobuf release page and select the compressed package file suitable for your operating system.
+
+For example, we can use the one from 'https://github.com/protocolbuffers/protobuf/releases/download/v21.7/protoc-21.7-osx-x86\_64.zip'
+
+unzip protoc-21.7-osx-x86\_64.zip and enter protoc-21.7-osx-x86\_64
+```
+$ cd protoc-3.20.3-osx-x86_64/bin
```
Comment (Abirdcfly) [inline observer-plugins/default-plugins/staging/github.com/kube-arbiter/arbiter/pkg/proto/README.md:11]: Maybe just one version is enough? 3.21.7 or 3.20.3

**Review by Abirdcfly — APPROVED** [review]
/lgtm

**Pull request merged**